db/migrate/20150826125744_add_storyline_id_to_chapters.rb

```diff
@@ -0,0 +1,6 @@
+class AddStorylineIdToChapters < ActiveRecord::Migration
+  def change
+    add_column :pageflow_chapters, :storyline_id, :integer
+    add_index :pageflow_chapters, [:storyline_id], name: "index_pageflow_chapters_on_storyline_id", using: :btree
```

Comment:
Line is too long. [113/100]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.